src/modules/modules.py

```diff
@@ -270,6 +276,36 @@ def forward(self, sent, task):
         word_mask = util.get_text_field_mask(sent).float()
         return word_embs, word_mask  # need to get # nonzero elts
 
+class PRPNPhraseLayer(Model):
+    ''' PRPN sentence encoder '''
```

Comment:
Try not to use 'sentence encoder' here, since this object isn't of type SentenceEncoder.

Reply:
(I know ON-LSTM does this below. Consider fixing that too, while you're at it.)